assets/js/signup.js

```diff
@@ -23,7 +11,7 @@ var loadQuestions = function() {
   });
 
   $.getJSON({
-    url: '/server/form.php?retrieve=questions',
+    url: '/server/form.php?retrieve=questions&subheadings[]=Contact Information&subheadings[]=Language Information&subheadings[]=Background Information',
```

Comment:
How would we handle needing to add a question or a subheading in the middle of the ordering? Or are we assuming the questions and subheading will never change?